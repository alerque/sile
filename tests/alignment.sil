\begin[papersize=a6]{document}
\nofolios
\script[src=packages/lorem]
\font[family=Libertinus Serif]
\set[parameter=document.parskip,value=10pt]

\raggedleft{\em{Ragged left} \lorem[words=40]}

\raggedright{\em{Ragged right} {\lorem[words=40]}}

\center{\em{Centered} \lorem[words=40]}

\justified{\em{Justified} \lorem[words=40]}

\rightalign{\em{Right aligned} \lorem[words=4]}

\end{document}
