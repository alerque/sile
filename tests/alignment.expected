Set paper size 	297.6377985	419.5275636
Begin page
Mx 	29.9059
My 	27.9564
Set font 	Libertinus Serif;10;400;Italic;normal;;LTR
T	51 66 72 72 70 69	(Ragged)
Mx 	62.9459
T	77 70 2373	(left)
Mx 	79.0159
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	77 80 83 70 78	(lorem)
Mx 	105.7059
T	74 81 84 86 78	(ipsum)
Mx 	133.7159
T	69 80 77 80 83	(dolor)
Mx 	158.2159
T	84 74 85	(sit)
Mx 	170.9859
T	66 78 70 85	(amet)
Mx 	194.0859
T	68 80 79 84 70 85 70 85 86 83	(consetetur)
Mx 	240.0159
T	84 66 69 74 81 84 68 74 79 72	(sadipscing)
Mx 	36.3059
My 	39.9564
T	70 77 74 85 83	(elitr)
Mx 	56.0059
T	84 70 69	(sed)
Mx 	72.5059
T	69 74 66 78	(diam)
Mx 	95.7459
T	79 80 79 86 78 90	(nonumy)
Mx 	132.9859
T	70 74 83 78 80 69	(eirmod)
Mx 	164.9559
T	85 70 78 81 80 83	(tempor)
Mx 	197.5059
T	74 79 87 74 69 86 79 85	(invidunt)
Mx 	235.2659
T	86 85	(ut)
Mx 	246.7359
T	77 66 67 80 83 70	(labore)
Mx 	275.1259
T	70 85	(et)
Mx 	28.8759
My 	51.9564
T	69 80 77 80 83 70	(dolore)
Mx 	57.7659
T	78 66 72 79 66	(magna)
Mx 	88.2259
T	66 77 74 82 86 90 66 78	(aliquyam)
Mx 	129.1059
T	70 83 66 85	(erat)
Mx 	148.0259
T	84 70 69	(sed)
Mx 	164.5259
T	69 74 66 78	(diam)
Mx 	187.7659
T	87 80 77 86 81 85 86 66	(voluptua)
Mx 	226.8759
T	66 85	(at)
Mx 	237.6059
T	87 70 83 80	(vero)
Mx 	258.6459
T	70 80 84	(eos)
Mx 	275.1259
T	70 85	(et)
Mx 	51.3659
My 	63.9564
T	66 68 68 86 84 66 78	(accusam)
Mx 	89.1759
T	70 85	(et)
Mx 	99.8059
T	75 86 84 85 80	(justo)
Mx 	122.9359
T	69 86 80	(duo)
Mx 	141.3459
T	69 80 77 80 83 70 84	(dolores)
Mx 	174.1359
T	70 85	(et)
Mx 	184.7659
T	70 66	(ea)
Mx 	196.8059
T	83 70 67 86 78	(rebum)
Mx 	226.0559
T	84 85 70 85	(stet)
Mx 	243.7459
T	68 77 74 85 66	(clita)
Mx 	264.1059
T	76 66 84 69	(kasd)
Mx 	240.6959
My 	75.9564
T	72 86 67 70 83 72 83 70 79	(gubergren)
Mx 	14.8819
My 	107.9564
Set font 	Libertinus Serif;10;400;Italic;normal;;LTR
T	51 66 72 72 70 69	(Ragged)
Mx 	47.9219
T	83 74 72 73 85	(right)
Mx 	70.8919
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	77 80 83 70 78	(lorem)
Mx 	97.5819
T	74 81 84 86 78	(ipsum)
Mx 	125.5919
T	69 80 77 80 83	(dolor)
Mx 	150.0919
T	84 74 85	(sit)
Mx 	162.8619
T	66 78 70 85	(amet)
Mx 	185.9619
T	68 80 79 84 70 85 70 85 86 83	(consetetur)
Mx 	231.8919
T	84 66 69 74 81 84 68 74 79 72	(sadipscing)
Mx 	14.8819
My 	119.9564
T	70 77 74 85 83	(elitr)
Mx 	34.5819
T	84 70 69	(sed)
Mx 	51.0819
T	69 74 66 78	(diam)
Mx 	74.3219
T	79 80 79 86 78 90	(nonumy)
Mx 	111.5619
T	70 74 83 78 80 69	(eirmod)
Mx 	143.5319
T	85 70 78 81 80 83	(tempor)
Mx 	176.0819
T	74 79 87 74 69 86 79 85	(invidunt)
Mx 	213.8419
T	86 85	(ut)
Mx 	225.3119
T	77 66 67 80 83 70	(labore)
Mx 	253.7019
T	70 85	(et)
Mx 	14.8819
My 	131.9564
T	69 80 77 80 83 70	(dolore)
Mx 	43.7719
T	78 66 72 79 66	(magna)
Mx 	74.2319
T	66 77 74 82 86 90 66 78	(aliquyam)
Mx 	115.1119
T	70 83 66 85	(erat)
Mx 	134.0319
T	84 70 69	(sed)
Mx 	150.5319
T	69 74 66 78	(diam)
Mx 	173.7719
T	87 80 77 86 81 85 86 66	(voluptua)
Mx 	212.8819
T	66 85	(at)
Mx 	223.6119
T	87 70 83 80	(vero)
Mx 	244.6519
T	70 80 84	(eos)
Mx 	261.1319
T	70 85	(et)
Mx 	14.8819
My 	143.9564
T	66 68 68 86 84 66 78	(accusam)
Mx 	52.6919
T	70 85	(et)
Mx 	63.3219
T	75 86 84 85 80	(justo)
Mx 	86.4519
T	69 86 80	(duo)
Mx 	104.8619
T	69 80 77 80 83 70 84	(dolores)
Mx 	137.6519
T	70 85	(et)
Mx 	148.2819
T	70 66	(ea)
Mx 	160.3219
T	83 70 67 86 78	(rebum)
Mx 	189.5719
T	84 85 70 85	(stet)
Mx 	207.2619
T	68 77 74 85 66	(clita)
Mx 	227.6219
T	76 66 84 69	(kasd)
Mx 	14.8819
My 	155.9564
T	72 86 67 70 83 72 83 70 79	(gubergren)
Mx 	16.9889
My 	187.9564
Set font 	Libertinus Serif;10;400;Italic;normal;;LTR
T	36 70 79 85 70 83 70 69	(Centered)
Mx 	57.2089
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	77 80 83 70 78	(lorem)
Mx 	83.8989
T	74 81 84 86 78	(ipsum)
Mx 	111.9089
T	69 80 77 80 83	(dolor)
Mx 	136.4089
T	84 74 85	(sit)
Mx 	149.1789
T	66 78 70 85	(amet)
Mx 	172.2789
T	68 80 79 84 70 85 70 85 86 83	(consetetur)
Mx 	218.2089
T	84 66 69 74 81 84 68 74 79 72	(sadipscing)
Mx 	263.9489
T	70 77 74 85 83	(elitr)
Mx 	20.9989
My 	199.9564
T	84 70 69	(sed)
Mx 	37.4989
T	69 74 66 78	(diam)
Mx 	60.7389
T	79 80 79 86 78 90	(nonumy)
Mx 	97.9789
T	70 74 83 78 80 69	(eirmod)
Mx 	129.9489
T	85 70 78 81 80 83	(tempor)
Mx 	162.4989
T	74 79 87 74 69 86 79 85	(invidunt)
Mx 	200.2589
T	86 85	(ut)
Mx 	211.7289
T	77 66 67 80 83 70	(labore)
Mx 	240.1189
T	70 85	(et)
Mx 	250.7489
T	69 80 77 80 83 70	(dolore)
Mx 	17.4189
My 	211.9564
T	78 66 72 79 66	(magna)
Mx 	47.8789
T	66 77 74 82 86 90 66 78	(aliquyam)
Mx 	88.7589
T	70 83 66 85	(erat)
Mx 	107.6789
T	84 70 69	(sed)
Mx 	124.1789
T	69 74 66 78	(diam)
Mx 	147.4189
T	87 80 77 86 81 85 86 66	(voluptua)
Mx 	186.5289
T	66 85	(at)
Mx 	197.2589
T	87 70 83 80	(vero)
Mx 	218.2989
T	70 80 84	(eos)
Mx 	234.7789
T	70 85	(et)
Mx 	245.4089
T	66 68 68 86 84 66 78	(accusam)
Mx 	29.4989
My 	223.9564
T	70 85	(et)
Mx 	40.1289
T	75 86 84 85 80	(justo)
Mx 	63.2589
T	69 86 80	(duo)
Mx 	81.6689
T	69 80 77 80 83 70 84	(dolores)
Mx 	114.4589
T	70 85	(et)
Mx 	125.0889
T	70 66	(ea)
Mx 	137.1289
T	83 70 67 86 78	(rebum)
Mx 	166.3789
T	84 85 70 85	(stet)
Mx 	184.0689
T	68 77 74 85 66	(clita)
Mx 	204.4289
T	76 66 84 69	(kasd)
Mx 	226.0789
T	72 86 67 70 83 72 83 70 79	(gubergren)
Mx 	34.8819
My 	255.9564
Set font 	Libertinus Serif;10;400;Italic;normal;;LTR
T	43 86 84 85 74 2349 70 69	(Justified)
Mx 	72.4725
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	77 80 83 70 78	(lorem)
Mx 	100.2530
T	74 81 84 86 78	(ipsum)
Mx 	129.3536
T	69 80 77 80 83	(dolor)
Mx 	154.9442
T	84 74 85	(sit)
Mx 	168.8048
T	66 78 70 85	(amet)
Mx 	192.9953
T	68 80 79 84 70 85 70 85 86 83	(consetetur)
Mx 	240.0159
T	84 66 69 74 81 84 68 74 79 72	(sadipscing)
Mx 	14.8819
My 	267.9564
T	70 77 74 85 83	(elitr)
Mx 	33.8353
T	84 70 69	(sed)
Mx 	49.5887
T	69 74 66 78	(diam)
Mx 	72.0821
T	79 80 79 86 78 90	(nonumy)
Mx 	108.5755
T	70 74 83 78 80 69	(eirmod)
Mx 	139.7989
T	85 70 78 81 80 83	(tempor)
Mx 	171.6023
T	74 79 87 74 69 86 79 85	(invidunt)
Mx 	208.6157
T	86 85	(ut)
Mx 	219.3391
T	77 66 67 80 83 70	(labore)
Mx 	246.9825
T	70 85	(et)
Mx 	256.8659
T	69 80 77 80 83 70	(dolore)
Mx 	14.8819
My 	279.9564
T	78 66 72 79 66	(magna)
Mx 	45.8493
T	66 77 74 82 86 90 66 78	(aliquyam)
Mx 	87.2367
T	70 83 66 85	(erat)
Mx 	106.6641
T	84 70 69	(sed)
Mx 	123.6715
T	69 74 66 78	(diam)
Mx 	147.4189
T	87 80 77 86 81 85 86 66	(voluptua)
Mx 	187.0363
T	66 85	(at)
Mx 	198.2737
T	87 70 83 80	(vero)
Mx 	219.8211
T	70 80 84	(eos)
Mx 	236.8085
T	70 85	(et)
Mx 	247.9459
T	66 68 68 86 84 66 78	(accusam)
Mx 	14.8819
My 	291.9564
T	70 85	(et)
Mx 	25.5155
T	75 86 84 85 80	(justo)
Mx 	48.6492
T	69 86 80	(duo)
Mx 	67.0628
T	69 80 77 80 83 70 84	(dolores)
Mx 	99.8565
T	70 85	(et)
Mx 	110.4901
T	70 66	(ea)
Mx 	122.5338
T	83 70 67 86 78	(rebum)
Mx 	151.7874
T	84 85 70 85	(stet)
Mx 	169.4811
T	68 77 74 85 66	(clita)
Mx 	189.8447
T	76 66 84 69	(kasd)
Mx 	211.4984
T	72 86 67 70 83 72 83 70 79	(gubergren)
Mx 	135.7259
My 	323.9564
Set font 	Libertinus Serif;10;400;Italic;normal;;LTR
T	51 74 72 73 85	(Right)
Mx 	160.8459
T	66 77 74 72 79 70 69	(aligned)
Mx 	193.7859
Set font 	Libertinus Serif;10;400;;normal;;LTR
T	77 80 83 70 78	(lorem)
Mx 	220.4759
T	74 81 84 86 78	(ipsum)
Mx 	248.4859
T	69 80 77 80 83	(dolor)
Mx 	272.9859
T	84 74 85	(sit)
End page
Finish
